packages/pipeline/src/scripts/pull_missing_blocks.ts

```diff
@@ -14,56 +14,56 @@ import { handleError, INFURA_ROOT_URL } from '../utils';
 // Number of blocks to save at once.
 const BATCH_SAVE_SIZE = 1000;
 // Maximum number of requests to send at once.
-const MAX_CONCURRENCY = 10;
+const MAX_CONCURRENCY = 20;
```

Comment:
I also increased the maximum number of concurrent requests. I've been monitoring memory and CPU usage and this should speed up the script without taxing resources too much.